forge-gui/res/cardsfolder/g/guard_dogs.txt

```diff
@@ -3,8 +3,9 @@ ManaCost:3 W
 Types:Creature Dog
 PT:2/2
 A:AB$ ChooseCard | Cost$ 2 W T | Choices$ Permanent.YouCtrl | Mandatory$ True | ChoiceTitle$ Choose a permanent you control | RememberChosen$ True | SubAbility$ DBPrevent | SpellDescription$ Choose a permanent you control. Prevent all combat damage target creature would deal this turn if it shares a color with that permanent.
-SVar:DBPrevent:DB$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature whose combat damage will be prevented | KW$ Prevent all combat damage that would be dealt by CARDNAME. | ConditionDefined$ Targeted | ConditionPresent$ Card.SharesColorWith Card.IsRemembered | ConditionCompare$ GE1 | SubAbility$ DBCleanup
-SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:DBPrevent:DB$ Effect | ValidTgts$ Creature | TgtPrompt$ Select target creature whose combat damage will be prevented | ReplacementEffects$ RPrevent | RememberObjects$ Targeted | ExileOnMoved$ Battlefield | ConditionDefined$ Targeted | ConditionPresent$ Card.SharesColorWith Card.IsRemembered | ConditionCompare$ GE1 | SubAbility$ DBCleanup
```

Comment:
Probably `SpellDescription$ Prevent all combat damage target creature would deal this turn if it shares a color with that permanent.` and cut that from line 5